apps/src/applab/applab.js

```diff
@@ -1213,6 +1219,11 @@ Applab.encodedFeedbackImage = '';
  * @param {ApplabInterfaceMode} mode
  */
 function onInterfaceModeChange(mode) {
+  Applab.setWorkspaceMode(mode);
+
+  var showDivApplab = (mode !== ApplabInterfaceMode.DESIGN);
+  Applab.toggleDivApplab(showDivApplab);
```

Comment:
I wonder if we're at a point where some of this logic (at least toggleDivApplab?) could be moved into React components? Or are we still dealing with protected divs for some of these?